l10n_co_res_partner/doc/index.rst

```diff
@@ -0,0 +1,25 @@
+=============
+l10n_co_res_partner
+=============
+
+Installation
+============
+
+Dependencies: account, account_accountant, base
+
+Once you've installed the module, you should be able to see the new partner form.
+As your existing contacts still won't have all data, your entire contact list will show you some red lines, saying you
+should update your data.
+
+Usage
+=====
+
+Once you start modifying your existing contacts you will enjoy the benefits of the module. Fill in all mandatory data
+and the visual hints will disappear.
+This module does NOT create any kind of reports.
+
+Uninstallation
+==============
+
+Once you uninstall the module, all your new data (in the new fields) will be lost.
+Data from the standard odoo fields won't be lost.
```

Comment:
If there is nothing technical to mention (which is already on readme). We do not need the doc section (if you want extend it , it is usefull have different kind of information to avoid duplicate the job to mantain 2 docs in the same module.)